recipes/erlang/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}"
```

Comment:
We should add a clause above that sets 32-bit or 64-bit support based on `$ARCH` value and then adds the flag `--enable-m32-build` or `--enable-m64-build` as needed.